### ISSUE TITLE
Bug: The request cannot be forwarded because there is no online partner

### ISSUE THREAD

**Issue by bakhshandeh** [muon-protocol/muon-node-js#355]
`{"success":false,"appId":"45810160343022089601028851206408642627529628329831086458846511772841706178168","error":{"message":"The request cannot be forwarded because there is no online partner","onRemoteSide":true}}`

This error occurs sometimes when requests are sent to a non-deployer node.
It happens on both ALICE and PION.

I stress tested the deployers. Direct requests to deployers is working fine.

**Comment by bakhshandeh** [comment]
When the network restarts, the rate of this error is higher and for a short period all requests fails.
The issue could be on delegates and connection between other nodes and deployers.

**Comment by sadeghte** [comment]
The reason is the ifSynced middleware:

```
{
  success: false,
  appId: "45810160343022089601028851206408642627529628329831086458846511772841706178168",
  error: {
    message: "cannot found 1 online from 3 candidate nodes.",
    responses: {
      2: "node context not synced yet",
      3: "node context not synced yet",
      67: "peer QmYNF1T7FskYQQk6REheuMJaH4EMVmBfmn24WfSpcPtsRC not found to check online status."
    },
    onRemoteSide: true
  }
}
```

**Comment by sadeghte** [comment]
fixed in [e52e5d5](https://github.com/muon-protocol/muon-node-js/commit/e52e5d5739718e940ea3b3301cccc817d2d84641)
